core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -746,6 +746,8 @@ public enum Property {
       "The number of threads used to delete RFiles and write-ahead logs", "1.3.5"),
   GC_TRASH_IGNORE("gc.trash.ignore", "false", PropertyType.BOOLEAN,
       "Do not use the Trash, even if it is configured.", "1.5.0"),
+  GC_TRASH_IGNORE_IMPORTS_ONLY("gc.trash.ignore.imports.only", "false", PropertyType.BOOLEAN,
+      "Skip trash only for Import files when gc.trash.ignore is true.", "2.1.0"),
```

Comment:
I think this will be since 3.0.0 (or maybe 2.1.1 if backported).

Reply:
Updated to show 3.0.0